src/App.js

```diff
@@ -39,37 +54,52 @@ const App = () => {
 
   return (
     <SettingsContext.Provider value={settingsState}>
+      <ThemeProvider theme={darkTheme}>
+
+        <div className="app">
+
+          <SplitPane
+            split="vertical"
+            size={350}
+            allowResize={false}
+            pane1ClassName="pane-1"
+            pane2ClassName="pane-2"
+          >
+
+            <div className="editor-settings">
+              <EditorPanel />
+            </div>
+
+
+            <div className="editor-overlay">
+
+              <div className="editor-overlay-preview">
+
+
+                <Overlay
+                  {...( {
+                    gurmukhi: GURMUKHI,
+                    larivaarGurbani: true,
```

Comment:
If you are always passing `true` through, why pass it at all? Refactor to get rid of it.

We can also observe the pattern of `englishTranslation: TRANSLATION_ENGLISH` seems a little repetitive. A way of improving this would be to refactor to a default export of this object in the source file, and spreading it here.

Reply:
How do I make sure that spreading the variables will assign to the correct prop?